scripts/dts/gen_defines.py

```diff
@@ -221,6 +221,7 @@ def write_props(node):
             out_dev(node, ident, 1 if prop.val else 0)
         elif prop.type == "string":
             out_dev_s(node, ident, prop.val)
+            out_dev(node, ident + "_UNSTRING", to_ident(prop.val))
```

Comment:
I think `_TOKEN` would be a better suffix as the desire is to have something that can act as a token in preprocessor manipulation, e.g. `DT_PROPERTY##_cfg`.

Reply:
Strings are tokens too, so think it might be confusing.

Reply:
I don't agree.  "unstring" is more likely to be confusing: it's weird and a term nobody uses.  String literals may be tokens, but they can't participate in token pasting, which is the entire reason why we need something distinct from the macro that provides the string value.

Reply:
`*_TOKEN` would've been very confusing to me at least.

I like `*_UNSTRING`. Tells you what it does. We kinda make up words as we go more than English though.~

Reply:
`*_UNQUOTED` might work too, if you'd prefer that.

Reply:
Changed it to `*_UNQUOTED`. Think I like it better than `*_UNSTRING`.